packages/stryker-karma-runner/src/KarmaConfigEditor.ts

```diff
@@ -3,48 +3,43 @@ import * as karma from 'karma';
 import { InputFileDescriptor } from 'stryker-api/core';
 import { ConfigEditor, Config as StrykerConfig } from 'stryker-api/config';
 import KarmaConfigReader from './KarmaConfigReader';
+import { KARMA_CONFIG, KARMA_CONFIG_FILE } from './configKeys';
 
 const log = log4js.getLogger('KarmaConfigEditor');
```

Comment:
We still set the logger outside of the class here

Reply:
Yes, let's fix that as well in this PR

Reply:
You fixed it in a different file 👍